pkg/cmd/utils.go

```diff
@@ -218,3 +219,19 @@ func FetchShootFromTarget(target TargetInterface) (*gardencorev1beta1.Shoot, err
 
 	return shoot, nil
 }
+
+//get server field value from kubeconfig
+func getServerValueFromKubeconfig(kubeconfigPath string, kubeconfigReader KubeconfigReader) string {
+	var kc Kubeconfig
+	if strings.Contains(kubeconfigPath, "~") {
+		kubeconfigPath = filepath.Clean(filepath.Join(HomeDir(), strings.Replace(kubeconfigPath, "~", "", 1)))
+	}
```

Comment:
I just meant to move the highlighted part to a utility function to not have that much duplicate code.
E.g. if you search for the string `"~", "", 1)))` within the project you get a lot of duplicate / similar code that could be extracted to some utility function
This is only a suggestion to make the code cleaner

Reply:
Ah ok, got your point.
I will move the function `getServerValueFromKubeconfig` back to `target.go`.

For refactor of these three lines of code to an function and clean whole project , i will do in another PR

Reply:
thanks